docs/src/examples/general_examples/basic_usage.jl

```diff
@@ -6,7 +6,7 @@ using SCS
 
 # ### Linear program
 #
-# $$
+# ```math
```

Comment:
I couldn't get `$$` working (even playing with the mathengine stuff), so switched all display math to this, following JuMP

Reply:
:+1:

Reply:
This is much clearer anyway

It might have to be `# $$\begin \n... \n# \end{array}$$`